### DISCUSSION TITLE
background noise remove

### DISCUSSION THREAD

**Discussion by jamesdesouza** [katspaugh/wavesurfer.js#2742] [Q&A] [answered]
Hello, 
I need to ask if it's possible to remove background noise from an mp3 file?

I've read there were some plugins for it but I can not found them.

**Comment by katspaugh** [comment, marked as answer]
You can do it via Web Audio as a set of bandpass filters at certain frequencies. E.g. https://github.com/ericschmidt/noise-reduction
Once you have the filters, you can insert them into the wavesurfer graph via `wavesurfer.setFilter`.

**Reply by jamesdesouza** [reply]
Thank you for your enlightening answer @katspaugh 
I'll try ASAP :)
If I found a solution, I will create a plugin for it.
I'm just wondering if it's already there are some plugins in order to noise reduction?

**Reply by katspaugh** [reply]
Not that I know of. Cheers.
